api/v1beta1/cluster_types.go

```diff
@@ -167,6 +172,11 @@ type MachineDeploymentTopology struct {
 	// +optional
 	Replicas *int32 `json:"replicas,omitempty"`
 
+	// MachineHealthCheck defines the control to enable, disable and override
```

Comment:
```suggestion
	// MachineHealthCheck allows to enable, disable and override
```
nit: let's use the same wording as above